book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+
+**Goals for Event Attendees:** 
+- Identify areas of research projects where wider ethical implications can be considered in different research contexts.
+- Learn through a hands-on workshop how to embed ethics and reproducibility into current work or research.
+- Discover what other attendees are working on in these areas.
+
+(er-datahazardscasestudy-organising)=
+## Organising the Event
+Here we discuss the logistics for organising an event, from setting up and promoting the event on an external platform to collaborating with volunteers and speakers to help successfully run the event on the day. 
+
+(er-datahazardscasestudy-organising-agenda)=
+### Creating an Agenda:
+The first step was to put together some initial ideas for the agenda.
+We worked together using Google Docs. 
+We used Google Docs because it allowed us to easily share the documents with other people, and it was easy to access.
+We made sure that no private or sensitive data was shared in the Google Docs. 
+Our finalised agenda is [here](https://docs.google.com/document/d/13buQgzAbTTSWVtONXC3cnjegWapJ0CBkL7TBkqNNRqU/edit).
+What did this process look like for us?
+ - brainstorming themed talks,
+ - deciding on keynote speakers, 
+ - practicalities of running the [Data Hazards Workshop](https://datahazards.com/contents/materials/workshop.html),
+ - identifying the symposium's target audience, 
+ - planning the event start and end times, including rest breaks*, 
+ - catering options,
+ - deciding whether to run a hybrid event (online and in-person). Guidelines for hybrid collaboration and events can be found [here](https://the-turing-way.netlify.app/collaboration/hybrid-collab).
+ 
+*To make the event more accessible, it was important for us to **make sure there were enough 'bio breaks'** to allow people to stretch their bodies, use the toilet and take a breather.
+
+**Speakers**
+A major part of putting together the agenda was all about picking our speakers. Speakers are what made our Symposium come alive, so we were eager to connect with different people to make sure our event had a mix of perspectives.
+
+When looking for speakers we:
+- reached out directly to keynote speakers whose research area or field considered data ethics and reproducibility.
+- invited speakers from both inside and outside our organisation to include a variety of experiences. 
+- asked for abstracts and titles, providing potential speakers with an estimated length of talk (including time for questions) and session they would present in.
+
+
+(er-datahazardscasestudy-organising-advertising)=
+### Advertising the Event
+
+**Choosing a date and gauging interest:** 
+Before even asking for funding, we wanted to know if there was enough interest to organise a symposium. So we reached out to our community *before beginning to organise* the symposium to gauge interest and spread the word. 
```

Comment:
These two sentences should be split on two lines.

Also, I'm curious what means you had to reach out to the community? How did you identify it, and so on?